fdbserver/IPager.cpp

```diff
@@ -0,0 +1,75 @@
+/*
+ * IPager.cpp
```

Comment:
Do we need this `.cpp` file, seem like the only thing it has is Unit test method. If `.h` doesn't allow unitest tests, can we add this unit test as part of other Redwood .cpp files?

Reply:
I don't know whether having unit test in the header file is a good idea. The VersionedBTree.actor.cpp file is already huge and has long compile time. I'm feeling over time we will be adding more unit test for the IPager.h code, so I add a .cpp file.

Reply:
I don't have any strong opinion about it, but yes it was just weird, hence, pointed out.